recipe/meta.yaml

```diff
@@ -94,55 +94,24 @@ outputs:
         - {{ pin_compatible(compiler('m2w64_fortran')) }}  # [win and cgo]
     test:
       requires:
-        - {{ compiler('c') }}         # [unix and cgo]
-        - {{ compiler('m2w64_c') }}   # [win and cgo]
+        - {{ compiler('c') }}        # [unix and cgo]
+        - binutils                   # [linux and cgp]
+        - {{ compiler('m2w64_c') }}  # [win and cgo]
         - git  # [linux]
         - perl
-      script: {{ go_variant_str }}/test.sh  # [unix]
-      script: {{ go_variant_str }}/test.bat  # [win]
+      files:
+        - {{ go_variant_str }}
+      commands:
+        - chmod +x {{ go_variant_str }}/test.sh   # [unix]
+        - ./{{ go_variant_str }}/test.sh   # [unix]
+        - {{ go_variant_str }}/test.bat  # [win]
     about:
       home: http://golang.org
       license: BSD-3-Clause
       license_family: BSD
       license_file: go/LICENSE
       summary: The Go Programming Language
 
-  - name: {{ name }}-{{ go_variant_str }}_{{ target_platform }}
-    script: compiler/build.sh  # [unix]
-    script: compiler/build.bat  # [win]
-    requirements:
-      run:
-        # Do this rather than `pin_subpackage(name, exact=True)` so targeting
-        # CGO to more than one version of a platform's C/C++/Fortran compilers
-        # is possible; an exact subpackage pin would (arbitrarily) select one
-        # version of the C/C++/Fortran compilers, and attempting to use other
-        # versions would result in unresolvable environment errors.  This would
-        # (likely) not be an issue if we addressed conda/conda-build#4142.
-        - {{ pin_subpackage(name, max_pin='x.x.x.x') }}
-        - _go_select  =={{ go_variant_ver }}={{ go_variant_str }}
-    #TODO: Re-enable once conda/conda#9845 if fixed.
-    #run_exports:   # [osx]
-    #  strong:                                     # [osx]
-    #    - __osx >={{ MACOSX_DEPLOYMENT_TARGET }}  # [osx]
-    test:
-      script: compiler/test.sh  # [unix]
-      script: compiler/test.bat  # [win]
-    about:
-      home: http://golang.org
-      license: BSD-3-Clause
-      license_family: BSD
-      license_file: go/LICENSE
-      summary: The Go ({{ go_variant_str }}) compiler activation scripts for conda-build.
-      description: |
-        This package enables the CONDA_GO_COMPILER environment variable. 
-        This variable is used by conda-forge's patched GoLang compiler to 
-        set conda-aware GOBIN and GOPATH default values. These default 
-        values are ignored if GOBIN and GOPATH environment variables 
-        are set independently by the user.
-
-        For more details, please see the go-feedstock recipe.
-      dev_url: https://github.com/conda-forge/go-feedstock
-
 about:
   home: http://golang.org
```

Comment:
These links should be updated:
https://go.dev/
https://go.dev/doc/
https://github.com/golang/